modules/common/math/mpc_osqp_test.cc

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/common/math/mpc_osqp.h"
+
+#include "cyber/common/log.h"
+#include "gtest/gtest.h"
+
+namespace apollo {
+namespace common {
+namespace math {
+
+TEST(MPCOSQPSolverTest, MPCOSQP) {
```

Comment:
Add test against null matrix and again non-full rank matrix

Reply:
Added. Thank you.